UbuntuDrivers/detect.py

```diff
@@ -189,6 +190,26 @@ def _apt_cache_modalias_map(apt_cache):
 
     return result2
 
+def package_get_nv_allowing_min_driver(did):
+    '''Get nvidia allowing driver for specific devices.
+
+    did: 0x1234
+    Return the situable nvidia driver version for it.
+    '''
+    path = "/etc/force_install.yaml"
+    min_version = None
+    with open(path, "r") as stream:
+        try:
+            gpus = list(yaml.safe_load(stream)['chips'])
+            for gpu in gpus:
+                if gpu['devid'] == did:
+                    min_version = gpu['minimumbranch']
+                    print("Found a specific nv driver version %s for %s(%s)" %
```

Comment:
You might want to use logging.debug() instead print(), since it is part of a library.

Reply:
indeed, I fixed it in [c8fd143](https://github.com/tseliot/ubuntu-drivers-common/pull/71/commits/c8fd143b890f9cb9f6017f4f14bb040258288288)